src/operations/blas2/gemv.hpp

```diff
@@ -80,6 +80,244 @@ SYCL_BLAS_INLINE void AddSetColumns<rhs_t>::adjust_access_displacement() {
   rhs_.adjust_access_displacement();
 }
 
+template <typename lhs_t, typename matrix_t, typename vector_t, bool is_transposed, int cache_line_size, int work_per_thread>
+SYCL_BLAS_INLINE
+Gemv<lhs_t, matrix_t, vector_t, is_transposed, cache_line_size, work_per_thread>::
+      Gemv(lhs_t &_l, matrix_t &_mat, vector_t &_vec,
+            typename vector_t::index_t &_WGs_per_NC,
+            typename vector_t::index_t &_WGs_per_C,
+            typename vector_t::index_t &_local_memory_size)
+      : lhs_(_l), // Result is stored in this
+        matrix_(_mat), // Input matrix
+        vector_(_vec), // Input vector
```

Comment:
Given that the name `x` still appear e.g in index variable names, maybe it would make sense to call the vector `x_` or `x_vec`, and same for the matrix `A_`? For `y` it's not really why since it's an intermediate buffer so `lhs_` works

Reply:
Renamed